api/src/main/java/uk/gov/hmcts/bar/api/data/model/FullRemission.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.bar.api.data.model;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.*;
+
+import javax.validation.constraints.Pattern;
+
+@Data
+@NoArgsConstructor
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class FullRemission extends PaymentInstructionRequest {
```

Comment:
can we just have one class for model and dto. do we really need to inherit from PaymentInstrucitonRequest?